app-editors/ghostwriter/ghostwriter-1.8.0.ebuild

```diff
@@ -0,0 +1,56 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit qmake-utils xdg-utils
+
+DESCRIPTION="Cross-platform, aesthetic, distraction-free markdown editor"
+HOMEPAGE="https://wereturtle.github.io/ghostwriter/"
+SRC_URI="https://github.com/wereturtle/ghostwriter/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="debug"
+
+BDEPEND="dev-qt/linguist-tools:5"
+
+RDEPEND="
+	app-text/hunspell
```

Comment:
binary is linking against `libhunspell-1.7.so.0`, hunspell has according subslots, so you'll have to trigger rebuild on subslot bumps using `:=`